test_environment.yml

```diff
@@ -0,0 +1,18 @@
+name: Qutip-Test-Env
+channels:
+- conda-forge
+dependencies:
+- python==3.7.3
```

Comment:
It looks like these requirements were taken from the qutip-notebook repository, which is fine, but the versions are very old. Let's leave them as is for this PR but open an issue to update the versions to more recent ones.

Reply:
Alright, I will open an issue after the PR is merged.